pkg/pillar/types/ifnametopci.go

```diff
@@ -211,6 +229,10 @@ func IoBundleToPci(log *base.LogObject, ib *IoBundle) (string, error) {
 		}
 	} else if ib.Ifname != "" {
 		var err error
+		if ib.Type == IoNetEthVF {
+			long, err = vfIfNameToPci(ib.Ifname)
+			return long, err
+		}
```

Comment:
The logic here looks misplaced, we do not go into `pciLongExists` check below and do not handle the case `ib.PciLong != ""` above.

Reply:
Question is whether an ib.PciLong would be the PCI address for the PF. If that is the case then it would make sense to also have the VF check in the ib.PciLong != "" above (or have common code to handle IoBetEthVF with ifname being set to do the lookup of the long for the VF.

Reply:
Yes. PciLong for PF type would be PCI address of Physical function itself. So I've changed it accordingly.

Reply:
as @giggsoff pointed out to me, we need to check for one of them. Thanks again, fixed it!